test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -695,18 +695,62 @@
     (mt/dataset times-mixed
       (testing "Can compare across dates, datetimes, and with timezones from a table"
         ;; these particular numbers are not important, just that we can compare between dates, datetimes, etc.
-        (is (= [25200 -8349 33549]
-               (->> (mt/run-mbql-query times
-                      {:fields [[:expression "tz,dt"]
-                                [:expression "tz,d"]
-                                [:expression "d,dt"]]
-                       :limit 1
-                       :expressions
-                       {"tz,dt" [:datetime-diff $dt_tz $dt :second]
-                        "tz,d"  [:datetime-diff $dt_tz $d :second]
-                        "d,dt"  [:datetime-diff $d $dt :second]}})
-                    (mt/formatted-rows [int int int])
-                    first)))))))
+        (mt/with-temporary-setting-values [driver/report-timezone "UTC"]
```

Comment:
This test didn't change other than making sure the report-timezone is "UTC". Helpful when you're testing locally and you've screwed up your report-timezone state.